src/patches/dense_patch_computation.cpp

```diff
@@ -147,15 +147,6 @@ bool merge_patches(
 }
```

Comment:
This file is where most of the changes are ... github doesn't even want to show it by default :joy:

Reply:
Mostly added debugging to exceptions and tried to make intents more clear with more detailed naming. Noting where there is a logic change